versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md

```diff
@@ -82,90 +82,17 @@ Synchronize files between Web Modeler and version control systems (VCS) and vice
 
 For automatic file synchronization, consider maintaining a secondary system of record for mapping Web Modeler projects to VCS repositories. This system also monitors the project-to-repository mapping and update timestamps.
 
-<!-- To be added once GA released:
-
-To listen to changes in Web Modeler, you currently need to implement a polling approach that compares the update dates with the last sync dates recorded. Use the `POST /api/beta/files/search` [endpoint](https://modeler.cloud.camunda.io/swagger-ui/index.html#/Files/searchFiles) with this payload to identify recently updated files:
-
-```json title="POST /api/beta/files/search"
-{
-  "filter": {
-    "projectId": "<PROJECT TO SYNC>",
-    "updated": "<LAST SYNC DATE>",
-  ],
-  "page": 0,
-  "size": 50
-}
-```
--->
-
-For real-time synchronization, employ a polling approach comparing update dates with last sync dates. Use the `POST /api/beta/files/search` [endpoint](https://modeler.cloud.camunda.io/swagger-ui/index.html#/Files/searchFiles) with the following payload to discover recently updated files, and compare the `updated` date with your last sync date:
-
-```json title="POST /api/beta/files/search"
-{
-  "filter": {
-    "projectId": "<PROJECT TO SYNC>"
-  ],
-  "page": 0,
-  "size": 50
-}
-```
-
-:::info
-Pagination is enforced for all listed `search` endpoints. Ensure you obtain all relevant pages.
-:::
-
-We work to replace this with a webhook or subscription approach. An alternate approach involves manually triggering synchronization or delegating synchronization triggers to other sources, such as the pipeline itself, creation of new branches, or pull/merge requests.
+To listen to changes, starting from 8.3 Web Modeler offers a polling approach that compares the update dates with the last sync dates recorded, see [the 8.3 documentation](/versioned_docs/version-8.3/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md#sync-files-with-version-control).
```

Comment:
```suggestion
Starting in 8.3, Web Modeler offers a polling approach that compares the update dates with the last sync dates recorded, see [the 8.3 documentation](/versioned_docs/version-8.3/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md#sync-files-with-version-control).
```

Reply:
@pepopowitz is this going to cause problems in the future? If we want to link across versions, should we? Or should we ask users to use the version picker in the navbar?

Reply:
I think this is the best way to approach this. 

Asking the user with text in the docs to pick a version from the navbar feels like putting a burden on them that I don't think is necessary, and it relies on them to interpret the instructions correctly.

Linkng to a specific version makes sense, based on the text, and at archival time it shouldn't be a problem: 

* When this version (8.2) is archived, this link will show up as broken. We'll have to fix it, by making it a fully qualified URL to `https://docs.camunda.io/8.3/...`. This seems like a thing we should have to do (preserve links), and it's very minor.
* Later, when 8.3 archives, anything at `docs.camunda.io/8.3/` will be redirected to `unsupported.docs.camunda.io/8.3/`, and the link from the 8.2 site will Just Work ™️ . 

I don't think there are a lot of times where we will _want_ to link across versions; but I think this is a good example of a reason to do it, and it's the correct approach.

Reply:
Ignore my suggestion here, approving and merging.